playbooks/appsemblerPlaybooks/ficus_enterprise.yml

```diff
@@ -96,3 +96,21 @@
     - { role: "xqueue", update_users: True }
     - role: "githubsshkeys"
       when: "cloud_provider == 'azure'"
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-0"
```

Comment:
@johnbaldwin The problem that we've here, is that the cron must run only on one server. That's the reason to split this and install it in a different way on each server.